libs/design-system/dialog/src/lib/notice/notice.component.scss

```diff
@@ -0,0 +1,53 @@
+:host {
+  --mdc-dialog-container-color: white;
+  display: block;
+  min-width: 20rem;
+  max-width: 29.5rem;
+
+  &:has(.action) {
+    --mat-dialog-actions-alignment: space-between;
+  }
+
+  .mat-mdc-dialog-title {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0;
+    margin-bottom: 1rem;
+
+    .title {
+      margin-top: 1.75rem;
+      margin-left: 2rem;
+    }
+
+    button[mat-icon-button] {
+      margin: 0.875rem;
+    }
+  }
+
+  .mat-mdc-dialog-title::before {
+    display: none;
+  }
+
+  .mat-mdc-dialog-content {
+    --mdc-dialog-supporting-text-tracking: 0.5px;
+    padding: 0 2rem;
+    font: var(--sys-body-large);
```

Comment:
Missing letter-spacing

Reply:
--mdc-dialog-supporting-text-tracking: 0.5px;
is letter spacing